src/Components/ADT3DSceneBuilder/Internal/Behaviors/BehaviorsForm.tsx

```diff
@@ -592,6 +592,16 @@ const SceneBehaviorsForm: React.FC<IADT3DSceneBuilderBehaviorFormProps> = ({
                                 className={commonPanelStyles.formTabContents}
                                 headerText={t('3dSceneBuilder.visualRulesTab')}
                                 itemKey={BehaviorPivot.visualRules}
+                                onRenderItemLink={(props, defaultRenderer) =>
```

Comment:
@pasanchMSFT  is this tab required? Why do we need a validity map for this tab?

Reply:
I agree, don't think we need validity map for this. Since this will spawn a form to edit the contents that will handle the validity of data.